modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/context/provider/FilterContextProvider.java

```diff
@@ -113,8 +114,8 @@ public Filter createContext(Message message) {
 				expressionVisitException.getMessage(),
 				expressionVisitException);
 		}
-		catch (InvalidFilterException invalidFilterException) {
-			throw invalidFilterException;
+		catch (WebApplicationException webApplicationException) {
```

Comment:
I am scared about an InvalidFilterException being thrown after this change? Maybe we should include a new catch block instead of replacing it? :thinking: